components/web/apache2-modules/mod_auth_mellon/patches/001-use_gnu_bin_path.patch

```diff
@@ -0,0 +1,24 @@
+Solaris grep supports limited regular expressions.
+Use GNU grep in mellon_create_metadata.sh script.
```

Comment:
Please add runtime dependency on GNU grep like this: https://github.com/OpenIndiana/oi-userland/blob/10f5892f9ab4d7cb51b52631b9f5b046b5400a97/components/archiver/zstd/zstd.p5m#L24
I think it won't be found automatically.